packages/paste-website/src/components/tokens-list/TokensListFilter.tsx

```diff
@@ -9,6 +9,8 @@ import {Grid, Column} from '@twilio-paste/grid';
 import {Select, Option} from '@twilio-paste/select';
 import {useUID} from '@twilio-paste/uid-library';
 
+import {SITE_MASTHEAD_HEIGHT} from '../../constants';
```

Comment:
question: why do we still need this constant? Previously the MASTHEAD was fixed positioned but the new one should scroll away (relative)

[edit] I see now that you kept it fixed.